x-pack/plugins/osquery/server/routes/pack/find_pack_route.ts

```diff
@@ -28,17 +30,10 @@ export const findPackRoute = (router: IRouter) => {
         version: API_VERSIONS.public.v1,
         validate: {
           request: {
-            query: schema.object(
-              {
-                page: schema.maybe(schema.number()),
-                pageSize: schema.maybe(schema.number()),
-                sort: schema.maybe(schema.string()),
-                sortOrder: schema.maybe(
-                  schema.oneOf([schema.literal('asc'), schema.literal('desc')])
-                ),
-              },
-              { unknowns: 'allow' }
```

Comment:
I saw this `{ unknowns: 'allow' }` option in some validation objects.

how does the new validation work? checking the `buildRouteValidation()`, it looks to me that now there should be an exact match, without additional keys in the query.

is there a possibility that there are some API calls that (sadly) add additional query params, and therefore some functionality will break because of this change?

if not, it's great to be more exact :)

Reply:
Hey, thanks for raising this 👍  I think you've got it right, I was also thinking about it but in the end I think that having a more strict API would be safer. All our tests are still passing, the only concern would be if the users use any additioanl params (but these should not be supported anyway).  However, maybe @patrykkopycinski remembers if there was a specific reason to use `{ unknowns: 'allow' }` .

Reply:
@patrykkopycinski Confirmed it's no longer needed 👍